packages/jupyter-chat/src/components/messages/footer.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { Box } from '@mui/material';
+import React from 'react';
+import {
+  IMessageFooterRegistry,
+  MessageFooterSectionProps
+} from '../../footers';
+
+/**
+ * The chat footer component properties.
+ */
+export interface IMessageFootersProps extends MessageFooterSectionProps {
+  /**
+   * The chat footer registry.
+   */
+  registry: IMessageFooterRegistry;
+}
```

Comment:
One more naming suggestion:

```suggestion
export interface MessageFooterProps extends MessageFooterSectionProps {
  /**
   * The chat footer registry.
   */
  registry: IMessageFooterRegistry;
}
```

Reply:
It does not fit with the naming convention, an interface must start with `I`.
Let's keep it that way for now, we'll change it to type later if it make more sense.